### PR TITLE
Add pagination to page

### PR DESCRIPTION
**Pull request by AreRex14**
The page will have previous and next link. 

For example, if we have 10 published posts, 5 posts will appear on the first main page and the next link will be activated for other posts. 

When the next link is clicked, the page will go to sig-fstm.github.io/page2 and so on...

### CODE REVIEW

**Comment by IceM4nn** [comment]
Have you try the pagination plugin? Is it works flawlessly without issue? Or does it need any further setup, gemfile, etc..?

File: _config.yml [hunk 1]
```diff
@@ -3,6 +3,13 @@ description: We are SiG FSTM a small community from KUIS. This website will be t
 permalink: /posts/:title/
 theme: jekyll-theme-hacker
 
+plugins:
+  - jekyll-paginate
```
Comment (kaitoAckerman) [inline _config.yml]: @AreRex14 remove plugins jekyll-paginate. It should works without it. Try dlu. lgi pun jekyll-paginate dh discontinue.

Comment (kaitoAckerman) [reply _config.yml]: refer [Jekyll paginate repo](https://github.com/jekyll/jekyll-paginate)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+---
+
+<h1 style="text-align:center">All posts</h1>
+
+{% for post in paginator.posts %}
+<h2>
+<a href="{{ post.url }}">{{ post.title }}</a>
```
Comment (kaitoAckerman) [inline index.html]: Yang ni tkyah bgi link. kalau tk nmpk title biru, read more button pun biru. Lagi pelik

**Comment by AreRex14** [comment]
@IceM4nn @kaitoAckerman sorry late, I have remove the code line and it works fine, I already test it...so good to go I think.

**Comment by AreRex14** [comment]
Please check the new pull request

**Comment by kaitoAckerman** [comment]
Thank you for the changes!

**Pull request merged**